<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskSpaceWidget</class>
 <widget class="QWidget" name="DiskSpaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>115</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">DiskSpaceWidget</string>
  </property>
  <layout class="QVBoxLayout" name="pVerticalLayout_">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QProgressBar" name="pProgressBar_">
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="::cppdevtk::gui::KSqueezedTextLabel" name="pLabel_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">C: 75 GB free of 100 GB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>::cppdevtk::gui::KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header location="global">cppdevtk/gui/k_squeezed_text_label.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
